src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -95,4 +95,16 @@
      * @return True if can be interrupted, false if not
      */
     boolean canBeInterrupted();
+
+    /**
+     * Resets this task.
+     */
+    void reset();
+
+    /**
+     * Gets the mutex bits. When the | operation result of two mutex bits is 0, they can run
+     * concurrently.
+     * @return The mutex bits which categorizes this task
+     */
+    int getMutexBits();
```

Comment:
I just stated it so that it's known. Best to poke @Zidane about exposing mutex bits - I have no problem with it personally.